packages/lodestar-cli/src/cmds/beacon/cmds/run/run.ts

```diff
@@ -34,7 +36,13 @@ export async function runHandler(options: IBeaconOptions): Promise<void> {
 
   const config = getBeaconConfig(options.preset, options.params);
   const libp2p = await createNodeJsLibp2p(peerId, options.network);
-  const logger = new WinstonLogger();
+  const loggerTransports = [
+    consoleTransport
+  ];
+  if(options.logFile) {
+    loggerTransports.push(fileTransport(path.join(options.rootDir, options.logFile)));
```

Comment:
What if the user specifies `options.logFile` as an absolute path?

Reply:
it won't work, but same goes for all path we use :(

Reply:
True